### PR TITLE
[wix-style-react] - <Sidebar/> - fix gradient for inner menu

### PR DESCRIPTION
**Pull request by miby-wix**
### 🔦 Summary

...

### ✅ Checklist

- [ ] 👨‍💻 API change is approved by the UI Infra Developers 
- [ ] 👨‍🎨 UX change is approved by the Design System UX 
- 📚 Change is documented
  - [ ] Story
  - [ ] API description
  - [ ] Cheatsheet
  - [ ] Other (explain)
- 🔬 Change is tested
  - [ ] Component
  - [ ] Visual test

### CODE REVIEW

File: packages/wix-style-react/src/Sidebar/Sidebar.js [hunk 1]
```diff
@@ -299,33 +247,19 @@ class Sidebar extends Component {
...
-                {this.state.onScreenChildren}
-              </div>
-              {this.state.isScrollbarDisplayed && (
-                <div
-                  className={gradientClasses}
-                  data-hook={dataHooks.scrollBarGradient}
-                />
-              )}
-            </div>
+            <SidebarContentWrapper
```
Comment (gendeld) [inline packages/wix-style-react/src/Sidebar/Sidebar.js:250]: Maybe add back the 
`dataHooks.onScreenChildrenContainer`
datahook

**Comment by WixBuildServer** [comment]
### 🤖 Hi, I'm #Janet!

  A deploy preview of your PR was successfully released 🖼.

  You can find it under one of the following URLs:
  * https://preview.wix-style-react.com/pr_6634

File: packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+import PropTypes from 'prop-types';
+import { st, classes } from './Sidebar.st.css';
+import { dataHooks, sidebarSkins } from './constants';
+
+/** A sidebar content children wrapper to add gradient  */
+export class SidebarContentWrapper extends Component {
+  static propTypes = {
+    skin: PropTypes.oneOf(['dark', 'light']),
+    containerDataHook: PropTypes.string,
+    containerClasses: PropTypes.arrayOf(PropTypes.string),
```
Comment (gendeld) [inline packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js:11]: Shouldn't this be `PropTypes.string` instead of an array?

Comment (miby-wix) [reply packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js:11]: but it's a list for classes...

File: packages/wix-style-react/src/Sidebar/Sidebar.js (hunk 1 above)
Comment (miby-wix) [reply packages/wix-style-react/src/Sidebar/Sidebar.js:250]: But I'm not using it... I don't think I should add a datahook that is never used...

File: packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js (hunk 2 above)
Comment (gendeld) [reply packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js:11]: https://github.com/wix/stylable/blob/master/packages/runtime/src/types.ts#L24
st takes in classes and concats them to a class string
something like:
input:
st(classes.button, { disabled: true })
output:
"myButton_button myButton_button_disabled"

Comment (gendeld) [reply packages/wix-style-react/src/Sidebar/SidebarContentWrapper.js:11]: More or less

**Review by gendeld — APPROVED** [review]

**Pull request merged**